crane-cloud/frontend
# Refactor user-side route

# Description
This refactors the user side routing eliminating the UserID in the route for safety and security reasons

## Type of change


## Trello Ticket ID

https://trello.com/c/O087MGNL

## How Can This Be Tested?
Pull this branch locally, and after login navigate, to your projects, apps, their metrics and settings pages to confirm the new route changes aren't creating any breaking changes.

## Checklist:


# Screenshots
Not Applicable
